pkg/cloudevents/transport/http/transport.go

```diff
@@ -44,11 +44,6 @@ type Transport struct {
 	// If nil, DefaultShutdownTimeout is used.
 	ShutdownTimeout *time.Duration
 
-	// Sending
-
-	// Client is the http client that will be used to send requests.
-	// If nil, the Transport will create a one.
-	Client *http.Client
```

Comment:
This would be a breaking semver change, can you rework the PR to not change the api?

Reply:
Done.